doc/script_commands.txt

```diff
@@ -4083,18 +4083,20 @@ Note: rid2name may not produce correct character names since rid means
 
 ---------------------------------------
 
+*message <account ID>,"<message>";
 *message "<character name>","<message>";
 
 That command will send a message to the chat window of the character 
-specified by name. The text will also appear above the head of that 
-character. It will not be seen by anyone else.
+specified by account ID or name. The text will also appear above the head
+of that character. It will not be seen by anyone else.
 
 ---------------------------------------
 
-*dispbottom "<message>";
+*dispbottom "<message>",{<color>,{<account ID>}};
```

Comment:
>Using attachrid instead, you can check whether the attach command was successful.

completely agree with this one
that means our command will be different than rathena
but no matter, the RRGGBB color is more important